test/multiarch_utils.go

```diff
@@ -33,10 +34,20 @@ const (
 	Registry
 )
 
+// return architecture of the cluster where test suites will be executed.
+// default value is similar to build architecture, TEST_RUNTIME_ARCH is used when test target cluster has another architecture
+func getTestArch() string {
+	val, ok := os.LookupEnv("TEST_RUNTIME_ARCH")
+	if ok {
+		return val
+	}
+	return runtime.GOARCH
```

Comment:
Defaulting to runtime.GOARCH would default to the runtime of where the tests binary runs, not the architecture of the cluster, right?

That might be fine, but could cause trouble as more development workstations move to arm.

Reply:
Some value should be default, and architecture similarity between dev workstation and dev/test/prod k8s cluster is still most valid case, I believe.
For arm workstation and, for instance, amd64 k8s test cluster `TEST_RUNTIME_ARCH` variable is here :)

Reply:
Anyway this approach is just to tune the tests in arch specific manner. Most of the tests will run on any architecture(s) as is

Reply:
Thanks! Yeah this should be fine for now. My initial worry is that this value would also include GOOS (linux, darwin, etc.) and break everyone running on Mac. Now we just have to wait for the arm Macs for that :)